personal-growth/becoming-a-mentor.md

```diff
@@ -0,0 +1,118 @@
+  needs a very tight feedback loop (i.e. you need to immediately see the results of your work), you
+  might find mentoring to be frustrating.
+
+You should keep in mind that being a mentor is not the only way to advance in Nebulab: there is
+always room for people to grow as individual contributors, and you shouldn't feel pressured to get
+into mentoring just because you think it's the only option.
+
+If you still want to become a mentor, with all the pros and cons that entails, read on!
+
+## How mentoring works
+
+The first step to become a mentor is to... talk to your mentor. :) They will help you figure out if
+mentoring is for you, if you are ready and where to start. One thing to keep in mind is that we try
+to have one mentor every 3/4 individual contributors, so there might not be space for new mentors
+even though you are a good fit. If that's the case, just be patient and wait for the next hires!
+
+As a mentor, you will be assigned up to 4 mentees and 3 projects that you will manage. Of course,
+this will happen gradually, so that you have the time to get comfortable with your new role and
+responsibilities. Usually, individual contributors who become mentors are assigned to the client
+and team they are already working with, since it allows them to practice their mentoring skills in
+an environment they already know.
+
+You will still hold regular 1:1s with a member of the leadership team, who will follow your progress
+and give you advice on how to up your mentoring game. Mentors need mentors too, after all!
+
+## Running one-on-ones
+
+The tricky thing about one-on-ones is there is no right way to do them: every mentor has their 
+style, which means two one-on-ones can be different even when they're done in the same company. As a
+mentor, you are encouraged to find your unique style and voice.
+
+With that said, there are some ground rules to follow for holding effective one-on-ones (i.e.
+one-one-ones where people say what they want to say rather than what the manager wants to hear):
+
+- **It's about them, not you.** Even though the mentor schedules them, a one-on-one is about the
+  mentee, not the mentor. Let them define the agenda or not have an agenda at all, and let them
+  talk about whatever they want.
+- **Be interested and show it.** Your main job during a one-on-one is to be genuinely interested in
+  what the other person is saying, and to dig deeper and deeper until you are sure all the topics
+  have been discussed exhaustively. 
+- **Do not reschedule.** You should avoid rescheduling a one-on-one like the plague. Of course,
+  we're humans and things happen, so if you absolutely need to, go ahead and reschedule the meeting
+  as close as possible to the original date. 
+- **Create accountability.** It is extremely important for every one-on-one where problems have been
+  raised to create an action plan. Make sure both you and your mentee are accountable for that plan,
+  and monitor its progress closely.
```

Comment:
I'd add

**Be explicit.** Set clear expectations both for daily work and for attending 1-1 meetings. They could not imagine what you are expecting from them. Having a clear communication and documentation is key.

**Create a real connection.** To being effective you should build a real relationship with your mentee in order to gain mutual benefit from this practice. Trust is key.

**Measure.** You could need to set clear metrics around goals that a person should meet, the more accurate metrics are the more easier it will be for everyone to understand how things are going.

Reply:
I think that `Be explicit` partially disagree with `It's about them, not you.`. IMO when you set an expectation, the mentee follows that, and it's not 100% free to set his own goals. But I generally agree with being explicit 😄.
Also, I think that Measure and Create accountability are saying the same thing.

Reply:
> I think that "Be explicit" partially disagree with "It's about them, not you."

I think they are not in contrast actually: even to say to people that "it's about them" you have to say that explicitly and not expecting them to know by themselves. This is the sense that I would like to express on that point, not sure if I did it though. 😄

Reply:
@kennyadsl "Be interested and show it" is quite similar to "Create a real connection" I think! For the other two, I think we would need a few more examples (also "Be explicit" probably is a general recommendation rather than being specific to 1:1s). Do you mind if for the time being we skip them and maybe add them later when we have a better idea of how to phrase them/where to put them?